tempesta_fw/http.c

```diff
@@ -3103,6 +3103,9 @@ tfw_http_req_process(TfwConn *conn, const TfwFsmData *data)
 		tfw_http_req_parse_drop(req, 400, "failed to parse request");
 		return TFW_BLOCK;
 	case TFW_POSTPONE:
+		if (off > 0)
+			if (tfw_http_chop_skb((TfwHttpMsg *)req, skb, off, 0))
+				return TFW_BLOCK;
```

Comment:
Good catch!

Reply:
Shouldn't we try to chop `curr_skb_trail` here too?

Reply:
Agree, last argument must be `curr_skb_trail`. Also the condition `if (off > 0)` is not really required,  `tfw_http_chop_skb()` perfectly handles the case, when both `off` and `trail` are `0`.

Reply:
Thanks.
Changed the code to use `curr_skb_trail` and to call `tfw_http_chop_skb()` unconditionally.